### ISSUE TITLE
Support for SameSite cookie flag.

### ISSUE THREAD

**Issue by anio** [psf/requests#5613]
Hi,

Have you any plan to expose `samesite` cookie flag?

**Comment by sigmavirus24** [comment]
We use the standard library which doesn't expose it but I'm working on [moreos](/sigmavirus24/moreos) which will have a cookie jar that's saner, more secure, and would expose this for Requests as a drop-in replacement

**Comment by anio** [comment]
You are right, but I think it's possible to add to newer python versions. I added it to my fork for my use case.

https://github.com/anio/requests/commit/ae838c9741e39d53dea780d72137dfc4ad048b04

**Comment by sigmavirus24** [comment]
Newer python versions aren't all we support and further, don't have a samesite attribute here
